### PR TITLE
Treat manual sshkey as object in array

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7933/

File: jujugui/static/gui/src/app/components/deployment-flow/sshkey/sshkey.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ class DeploymentSSHKey extends React.Component {
       const manualKey = this.refs.sshKey.getValue();
       const key = this._validateAndSplitKey(manualKey);
       if (key) {
-        this.props.setSSHKeys(manualKey);
+        this.props.setSSHKeys([{text: manualKey}]);
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/deployment-flow/sshkey/sshkey.js]: shouldn't this be `setSSHKeys([key])`?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7934/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7935/

**Comment by Lukewh** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**